src/Atmos/Model/bc_energy.jl

```diff
@@ -114,3 +114,68 @@ function atmos_energy_normal_boundary_flux_second_order!(
     # we want to prescribe the inward flux
     fluxᵀn.ρe -= bc_energy.fn(state⁻, aux⁻, t)
 end
+
+"""
+    BulkFormulaEnergy(fn) :: EnergyBC
+
+Calculate the net inward energy flux across the boundary.
+The drag coefficient is `C_q = fn_C_q(state, aux, t, normPu_int)`.
+The surface temp is `T_sfc= fn_T_sfc(state, aus, t)`.
+The surface q_tot is `q_tot_sfc = fn_q_tot_sfc(state, aux, t)`.
+with signature `fn(state, aux, t)`, returning the flux (in W m^-2).
+"""
+struct BulkFormulaEnergy{FNX,FNT,FNM} <: EnergyBC
+    fn_C_h::FNX
+    fn_T_sfc::FNT
+    fn_q_tot_sfc::FNM
+end
+function atmos_energy_boundary_state!(
+    nf,
+    bc_energy::BulkFormulaEnergy,
+    atmos,
+    args...,
+) 
+end
+function atmos_energy_normal_boundary_flux_second_order!(
+    nf,
+    bc_energy::BulkFormulaEnergy,
+    atmos,
+    fluxᵀn,
+    n⁻,
+    state_sfc⁻,
+    diff_sfc⁻,
+    hyperdiff_sfc⁻,
+    aux_sfc⁻,
+    state_sfc⁺,
+    diff_sfc⁺,
+    hyperdiff_sfc⁺,
+    aux_sfc⁺,
+    bctype,
+    t,
+    state_int⁻,
+    diff_int⁻,
+    aux_int⁻,
+)
+
+    u_int⁻ = state_int⁻.ρu / state_int⁻.ρ
+    Pu_int⁻ = projection_tangential(atmos, aux_int⁻, u_int⁻)
```

Comment:
```suggestion
    u_int⁻_tan = projection_tangential(atmos, aux_int⁻, u_int⁻)
```
(or something like this, here and elsewhere, to be more descriptive)?